sepolicy/file_contexts

```diff
@@ -112,4 +107,14 @@
 
 /sys/kernel/debug/rmt_storage(/.*)? u:object_r:debugfs_rmt_storage:s0
 
-/data/vendor/nfc(/.*)? u:object_r:nfc_data_file:s0
+/(vendor|system/vendor)/bin/hw/android\.hardware\.health@2\.0-service\.msm8937        u:object_r:hal_health_default_exec:s0
+/(vendor|system/vendor)/bin/hw/android\.hardware\.vibrator@1\.0-service.msm8937   u:object_r:hal_vibrator_default_exec:s0
+/sys/devices/virtual/timed_output/vibrator/vtg_default                            u:object_r:sysfs_vibrator:s0
+/sys/devices/virtual/timed_output/vibrator/vtg_level                            u:object_r:sysfs_vibrator:s0
+/sys/devices/virtual/timed_output/vibrator/vtg_min                            u:object_r:sysfs_vibrator:s0
+/sys/devices/virtual/timed_output/vibrator/vtg_max                            u:object_r:sysfs_vibrator:s0
```

Comment:
Required for vibration?

Reply:
for the new HAL it seems like it is required